element-hq/element-x-android
# Account deactivation.

## Content


Add ability for the user to delete the account in-app, if this is a password based account.

## Motivation and context


Now the the application is able to create account, we need to provide a way to delete it.

## Screenshots / GIFs



![AccountDeactivation](https://github.com/user-attachments/assets/0d60169e-f09a-4153-888d-e8ea1cab3a6c)

Notes:
- I do not know why the first icon in the list is not tinted, probably a cache issue. Let's see how the screanshots are getting recorded.
- the "Deactivate" button in the confirmation dialog is now red (destructive)

## Tests



- Create an account (can be done in-app on matrix.org)
- Navigate to the setting a deactivate the account

Observe that the account has been deleted (this is not possible to login again) and the application is back to fresh install state.

## Tested devices

- OS version(s):

## Checklist
